R/cf.R

```diff
@@ -166,7 +166,7 @@ cf_principal_correlator <- function (.cf = cf(), id, gevp_reference_time) {
   .cf$id <- id
   .cf$gevp_reference_time <- gevp_reference_time
 
-  class(.cf) <- append(class(.cf), 'cf_principal_correlators')
```

Comment:
good catch @urbach

Reply:
@martin-ueding do you agree with this change? since the mixin-constructor is singular, the name of the mixin should be too, right?